draft-ietf-quic-recovery.md

```diff
@@ -1783,7 +1788,7 @@ OnCongestionEvent(sent_time):
 
   // Enter recovery period.
   congestion_recovery_start_time = now()
-  ssthresh = congestion_window * kLossReductionFactor
+  ssthresh = congestion_window * (1 - kLossReductionFactor)
```

Comment:
I tried to triple check this, but there is not much to compare against.
If the loss reduction factor is supposed to indicate the amount of **reduction**, we need this change.  When the value is 0.5 it doesn't matter, of course...

Reply:
The factor is multiplicative in cubic (0.7 there), so 1-B is likely to be problematic.

Reply:
So we should change the definition of kLossReductionFactor to be something like "scaling factor applied to reduce the congestion window when loss is detected"?